### PR TITLE
CLI: Use Svelte-native CSF during initialization

### PR DESCRIPTION
**Pull request by tooppaaa**
Issue:

## What I did

Use CSF in svelte CLI

## How to test

CLI

### CODE REVIEW

**Review by shilman — APPROVED** [review]
✅ Tested locally -- looks great

@j3rem1e can you review?

File: lib/cli/src/frameworks/svelte/Button.stories.svelte [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    backgroundColor: { control: "color" },
+    size: {
+      control: { type: "select", options: ["small", "medium", "large"] },
+    },
+    onClick: { action: "onClick" },
+  }}
+/>
+
+<Template let:args>
+  <Button {...args} on:click={args.onClick} />
```
Comment (j3rem1e) [inline lib/cli/src/frameworks/svelte/Button.stories.svelte:21]: I have seen this in a lot of examples in Storybook, but what is the purpose of declaring events as properties ? It's look like a React convention.

In Svelte, event and properties are not the same thing. It's not a big deal, but passing an undeclared property to a component in svelte will display a warning in development mode.

**Review by j3rem1e — APPROVED** [review]
It's unrelated to this change, but the 'Button.svelte' generated by the CLI is bugged. 

It works only if the story is fully rerendered, not if new properties are injected to a running component (`mode` and `style` internal properties are not reactive)

File: lib/cli/src/frameworks/svelte/Button.stories.svelte (hunk 1 above)
Comment (shilman) [reply lib/cli/src/frameworks/svelte/Button.stories.svelte:21]: Args can be properties, events, slots, etc.

The two main patterns for how we use them, tho we may develop more:
1) auto-generated controls to manipulate the value in UI
2) dependency injection, e.g. auto-generated actions, which makes the stories more portable (no direct dependency on `@storybook/addon-actions`

It's the latter case that's highlighted here

Comment (j3rem1e) [reply lib/cli/src/frameworks/svelte/Button.stories.svelte:21]: Thanks for the clarification. 

Something should probably be added to the plugin in order to remove this warning. maybe filtering actions, and subscribing automatically to the linked event.

**Pull request merged**